app/src/online/java/com/kamron/pogoiv/CrashlyticsWrapper.java

```diff
@@ -30,10 +26,16 @@ public static void init(Context context) {
 
     public static class CrashReportingTree extends Timber.Tree {
 
+        public final Context context;
+
+        public CrashReportingTree(Context context) {
+            this.context = context;
+        }
+
         @Override
         protected void log(int priority, String tag, String message, Throwable t) {
 
-            if (BuildConfig.isInternetAvailable && GoIVSettings.getSettings(mContext).getSendCrashReports()) {
+            if (BuildConfig.isInternetAvailable && GoIVSettings.getSettings(context).getSendCrashReports()) {
```

Comment:
I'd all about single focused PRs 😄 I'd prefer to merge this and tackle settings as a separate issue